vendor/bat-native-ads/src/bat/ads/internal/ads/serving/eligible_ads/pipelines/new_tab_page_ads/eligible_new_tab_page_ads_factory.cc

```diff
@@ -28,6 +28,11 @@ std::unique_ptr<EligibleAdsBase> EligibleAdsFactory::Build(
                                              anti_targeting_resource);
     }
 
+    case 3: {
+      return std::make_unique<EligibleAdsV2>(subdivision_targeting,
```

Comment:
Just to doublecheck, this is suppose to be `V2` correct?

Reply:
V3 as above, thanks